doc/mgmt/OpenConfig_Interfaces.md

```diff
@@ -0,0 +1,199 @@
+  }
+
+  identity SPEED_800GB {
+    base ETHERNET_SPEED;
+    description "800 Gbps Ethernet";
+  }
+  ```
+### 3.3.2 REST API Support
+#### 3.3.2.1 GET
+Supported
+```
+curl -X GET -k "https://100.94.113.103/restconf/data/openconfig-interfaces:interfaces/interface=Ethernet104" -H "accept: application/yang-data+json"
+{"openconfig-interfaces:interface":[{"config":{"enabled":true,"mtu":9100,"name":"Ethernet104"},"openconfig-if-ethernet:ethernet":{"config":{"auto-negotiate":false,"port-speed":"openconfig-if-ethernet:SPEED_10GB"},"state":{"auto-negotiate":false,"port-speed":"openconfig-if-ethernet:SPEED_10GB"}},"name":"Ethernet104","state":{"admin-status":"UP","description":"","enabled":true,"mtu":9100,"name":"Ethernet104"},"subinterfaces":{"subinterface":[{"config":{"index":0},"index":0,"openconfig-if-ip:ipv6":{"config":{"enabled":false}},"state":{"index":0}}]}}]}
+```
+```
+admin@sonic:~$ curl -X GET -k "https://100.94.113.103/restconf/data/openconfig-interfaces:interfaces/interface=Ethernet104" -H "accept: application/yang-data+json"
+{"openconfig-interfaces:interface":[{"config":{"description":"patch_leaf","enabled":true,"mtu":9150,"name":"Ethernet104"},"openconfig-if-ethernet:ethernet":{"config":{"auto-negotiate":true,"port-speed":"openconfig-if-ethernet:SPEED_10GB"},"state":{"auto-negotiate":true,"port-speed":"openconfig-if-ethernet:SPEED_10GB"}},"name":"Ethernet104","state":{"admin-status":"UP","description":"patch_leaf","enabled":true,"mtu":9150,"name":"Ethernet104"},"subinterfaces":{"subinterface":[{"config":{"index":0},"index":0,"openconfig-if-ip:ipv4":{"addresses":{"address":[{"config":{"ip":"10.0.0.248","prefix-length":31},"ip":"10.0.0.248","state":{"ip":"10.0.0.248","prefix-length":31}}]}},"openconfig-if-ip:ipv6":{"config":{"enabled":false}},"state":{"index":0}}]}}]}
+```
+#### 3.3.2.2 SET
+Supported
+#### 3.3.2.3 DELETE
+Supported
+### 3.3.3 gNMI Support
+#### 3.3.3.1 GET
+Supported
+#### 3.3.3.2 SET
+Supported
+#### 3.3.3.3 DELETE
+Supported
+# 4 Flow Diagrams
+Mapping attributes between OpenConfig YANG and SONiC YANG:
+|   OpenConfig YANG       |    Sonic-port YANG    |
+|-------------------------|-----------------------|
+|   name                  |      name             |
+|   auto-negotiate        |      autoneg          |
+|   port-speed            |      speed            |
+|   description           |      description      |
+|   mtu                   |      mtu              |
+|   enabled               |      admin\_status    |
+|   index                 |      index            |
+
+|   OpenConfig YANG       |    Sonic-interface YANG |
+|-------------------------|-------------------------|
+|   name                  |      name               |
+|   enabled               |      ipv6\_use\_link\_local\_only          |
+|   index                 |      index              |
+|   prefix-length         |      ip-prefix          |
+
+# 5 Error Handling
+Configuration for port-speeds not present in YANG will report an error.

```

Comment:
We can generalize this statement instead of just talking about port-speed alone.